node/ReleaseAndStage/ReadMe.md

```diff
@@ -1 +1,4 @@
-NodeJS based pipeline which creates a new version then builds and deploys the project into the Nexus repository
+NodeJS based pipeline which:
```

Comment:
this isn't nodejs based pipeline :),  rather it is a pipeline that is **for** nodejs applications.